cli/lib/kontena/client.rb

```diff
@@ -528,14 +535,16 @@ def ignore_ssl_errors?
     def handle_error_response(response)
       data = parse_response(response)
 
+      request_path = (last_request && last_request[:path] ? " (#{last_request[:path]})" : '')
```

Comment:
It looks like there is a `response.path` that you can use, avoiding the need for the `@last_request`.